ledger-service/http-json/src/it/scala/http/HttpServiceIntegrationTest.scala

```diff
@@ -269,6 +270,39 @@ abstract class HttpServiceIntegrationTest
     }
   }
 
+  "query POST with empty query" - {
+    "with an interface ID" in withHttpService { fixture =>
+      import com.daml.http.json.JsonProtocol._
+      for {
+        _ <- uploadPackage(fixture)(ciouDar)
+        aliceH <- fixture.getUniquePartyAndAuthHeaders("Alice")
+        (alice, aliceHeaders) = aliceH
+        _ <- postCreateCommand(
+          iouCommand(alice, CIou.CIou),
+          fixture,
+          aliceHeaders,
+        )
+
+        searchResp <- search(
+          List.empty,
+          Map(
+            "templateIds" -> Seq(TpId.IIou.IIou).toJson,
+            "query" -> spray.json.JsObject(),
+          ).toJson.asJsObject,
+          fixture,
+          aliceHeaders,
+        )
+      } yield inside(searchResp) {
+        case domain.OkResponse(acl, None, StatusCodes.OK) => {
+          discard { acl.size shouldBe 1 }
+          discard { acl.head.templateId shouldBe TpId.IIou.IIou.copy(packageId = acl.head.templateId.packageId) }
+          acl.head.payload shouldBe spray.json.JsObject()
+        }
+      }
+    }
```

Comment:
This test should go under the existing `"query POST with empty query"` category in `AbstractHttpServiceIntegrationTestTokenIndependent`. That way, it will be run separately under postgres and oracle configurations as well. As it happens, flipping the expected response of `fails given interface ID` to the new success response should be sufficient.

Reply:
I flipped it but found that a create was needed, thanks will move back to `AbstractHttpServiceIntegrationTestTokenIndependent`

Reply:
Moved it.